src/main/kotlin/no/ccat/utils/DevHarvestMock.kt

```diff
@@ -18,16 +18,18 @@ fun startMockServer() {
         mockserver.stubFor(get(urlEqualTo("/api/registration/apis"))
                 .willReturn(okJson(File("src/test/resources/contract/api-reg.json").readText())))
 
-        mockserver.stubFor(get(urlEqualTo("/api/publishers/12345678"))
+        mockserver.stubFor(get(urlEqualTo("/api/organizations/12345678"))
```

Comment:
skal ikke 'api' fjernes i tillegg?

Reply:
Nei, har gjort det sånn for å slippe alt for mange endringer i test-variabler